.github/workflows/openfortivpn.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Legacy
+
+on:
+  push:
+    branches-ignore: [master]
+    # Remove the line above to run when pushing to master
```

Comment:
Why not running tests when pushing commits to `master`? It can be a good information to know whether tests pass or not, after adding a commit manually.

Reply:
I don't know, just re-used Github's Super-Linter file. Perhaps because we're not supposed to push directly to master in the first place?

Reply:
We're not *supposed* to, but it could happen on exceptional circumstances :)
In such situations, why not doing the default (run tests) like for other branches?

Reply:
My guess is that force pushing to master happens only when we want to bypass the tests - as is currently the case with Travis CI.

Reply:
@adrienverge Now I know why we should ignore the `master` branch: we end up running the tests twice on every pull request, when submitting the pull requests and when merging the pull request into the `master` branch. That's a waste of resources.

I cannot find a way to distinguish between pushing a pull request to `master` and an exceptional direct push to `master`. I plan on restoring `branches-ignore: [master]`.